.circleci/config.yml

```diff
@@ -129,23 +132,24 @@ jobs:
           command: |
             choco install make
             choco install mingw
-      - run:
-          name: "Build C-KZG-4844"
-          command: |
-            cd c-kzg-4844/src
-            make blst
-            make CLANG_EXECUTABLE=gcc lib
+      - build_blst
       - run:
           shell: bash.exe
           name: "Build and test java binding"
           command: |
+            echo "Building and testing for mainnet preset"
             cd c-kzg-4844/bindings/java
-            make CLANG_EXECUTABLE=gcc build
+            make build -B
```

Comment:
maybe set `PRESET` to `mainnet` above, so it's clear without some unknown defaults
also I'd set it in this block (If I'm correct on its purpose)
https://github.com/ConsenSys/jc-kzg-4844/blob/3330f841e3304ea9b2b52b7007996d0773550fc5/.circleci/config.yml#L49

Reply:
The windows run is a bit weird because there was a bug with using make for testing with gradle because it was never exiting, so as a workaround I explicitly call `./gradlew test`. Anyways, I added PRESET=mainnet everywhere to be more specific. A good idea! Makes it more understandable.